scopesim/utils.py

```diff
@@ -414,38 +418,83 @@ def set_logger_level(which="console", level="ERROR"):
             hdlr.setLevel(level)
 
 
-def bug_report():
-    """Get versions of dependencies for inclusion in bug report."""
-    try:
-        from importlib import import_module
-    except ImportError:
-        import_module = __import__
+def _get_required_packages():
+    reqs = metadata.requires(__package__)
+    for req in reqs:
+        # Only include non-extra packages
+        if "extra" in req:
+            continue
+
+        name = req.split(">", maxsplit=1)[0].strip()
+        yield name
+
+
+def _get_all_irdb_pkgs(root: Path):
+    return [pkg_path for pkg_path in root.iterdir() if pkg_path.is_dir()
+            and not pkg_path.name.startswith("__")]
```

Comment:
I was wondering whether I could still comment wit a suggestion on an already merged PR. Apparently that is not possible (which makes sense). Nevertheless, we could make this one liner a bit more complex (untested):

```python
def _get_all_irdb_pkgs(root: Path):
    return [pkg_path for pkg_path in root.iterdir() if pkg_path.is_dir()
            and not pkg_path.name.startswith("__")] if root.is_dir() else []
```